src/execution_plan.h

```diff
@@ -260,6 +260,7 @@ typedef struct FlatExecutionPlan{
     FlatBasicStep onRegisteredStep;
     FlatBasicStep onUnpausedStep;
     long long executionMaxIdleTime;
+    bool registered;
```

Comment:
Lets make it execution flags, might be more of those in the future

Reply:
ok
still keeping values as true / false